include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -606,11 +613,13 @@ class ADIOS2IOHandler : public AbstractIOHandler
 
 #if openPMD_HAVE_MPI
 
-    ADIOS2IOHandler( std::string path, AccessType, MPI_Comm );
+    ADIOS2IOHandler( std::string path, AccessType, MPI_Comm = MPI_COMM_WORLD );
```

Comment:
Why default to `MPI_COMM_WORLD`?
I think that's dangerous, maybe `MPI_COMM_SELF` or even no default.